docs/source/user/invocation.rst

```diff
@@ -92,8 +92,7 @@ And you should see something like:
       -q, --quiet           Report only file names, or nothing. This option is
                             repeatable.
       --count               Print total number of errors and warnings to standard
-                            error and set the exit code to 1 if total is not
-                            empty.
+                            error after after all other output.
       --diff                Report changes only within line number ranges in the
```

Comment:
this file isn't meant to be kept up to date

Reply:
Hmm, fair enough, I had just found the same string in both places anyway and edited it by default. I can back out that change, give me a second.